src/servidor/ServidorService.ts

```
@@ -31,6 +31,12 @@ class ServidorService extends BaseService {
   async remove(id: number): Promise<void> {
     return this.delete(`/${id}`)
   }
+
+  async listar(): Promise<Servidor[]> {
+      const response: AxiosResponse<Servidor[]> = await this.get()
```

ESLint

[error] Delete `··`

(prettier/prettier)